cmd/testrunner.go

```diff
@@ -189,12 +189,7 @@ func testTypeCommandActionFactory(runner testrunner.TestRunner) cobraext.Command
 			return fmt.Errorf("cannot determine if package has data streams: %w", err)
 		}
 
-		// Temporarily until independent Elastic Agents are enabled by default,
-		// enable independent Elastic Agents if package defines that requires root privileges
-		runIndependentElasticAgent := manifest.Agent.Privileges.Root
-
-		// If the environment variable is present, it always has preference over the root
-		// privileges value (if any) defined in the manifest file
+		runIndependentElasticAgent := false
 		v, ok := os.LookupEnv(enableIndependentAgents)
```

Comment:
Do we need to check the environment variable here? We are also checking it in the runner now.

Reply:
Yes, it could be removed.